app/controllers/evms_controller.rb

```diff
@@ -63,11 +68,11 @@ def index
     # export
     respond_to do |format|
       format.html
-      format.csv {
+      format.csv do
         send_data(@project_evm.to_csv,
-                  :type => 'text/csv; header=present',
-                  :filename => 'evm_' + @project.name + '_' +Time.now.to_date.to_s + '.csv')
-      }
+                  type: 'text/csv; header=present',
+                  filename: 'evm_' + @project.name + '_' + Time.now.to_date.to_s + '.csv')
```

Comment:
__[rubocop]__
* Line is too long. [90/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)